docs/serving/samples/hello-world/helloworld-scala/index.md

```diff
@@ -5,4 +5,209 @@ weight: 1
 type: "docs"
 ---
 
-{{% readfile file="README.md" %}}
```

Comment:
Why remove this?

Reply:
Hi, @markusthoemmes, this is because we are using the `index.md` file instead to be able to use tabs. Sorry for late reply 😞